jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricherTest.java

```diff
@@ -65,16 +64,20 @@ private void setupExpectations(Map<String, Object> bootableJarConfig, Map<String
         ProcessorConfig c = new ProcessorConfig(null, null, jkubeConfig);
         Configuration.ConfigurationBuilder configBuilder = Configuration.builder();
         configBuilder.processorConfig(c);
-        when(project.getPlugins()).thenReturn(lst);
-        when(context.getProject()).thenReturn(project);
-        when(context.getConfiguration()).thenReturn(configBuilder.build());
+        context = context.toBuilder()
+                .project(project.toBuilder()
```

Comment:
Rather than creating a builder instance from member field `project` you should create it from context itself so that you don't lose changes done in `context.getProject()` after test setup
```suggestion
                .project(context.getProject().toBuilder()
```